script/cibuild

```diff
@@ -2,3 +2,4 @@
 set -e # halt script on error
 
 bundle exec htmlproofer .
+eslint js/*
```

Comment:
Actually it looks like `eslint` [only considers `.js` extensions by default](http://eslint.org/docs/user-guide/command-line-interface#--ext) so we are ok herem